deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -115,13 +119,44 @@ extern "C" __declspec(noreturn) void __fastfail(unsigned int);
 #define TAEF_FAST_FAIL System::Environment::FailFast(nullptr)
 #endif // #if !defined(__cplusplus_cli)
 
+#if defined(__clang__)
+#include <array>
+#endif
+
 /// <summary>
 /// The WEX namespace contains classes used for marking up C++ Tests.
 /// </summary>
 namespace WEX
 {
     namespace Private
     {
+        #if defined(__clang__)
+        // tempalated struct using variadic template args to represent a sequence of characters
+        template <unsigned... CharacterIndices> 
+        struct index_sequence{};
```

Comment:
Using std::index_sequence with the #include <utility> wrapped in a clang #if def sounds good to me.